### PR TITLE
Add kedro-mlflow==0.8.1

### PR DESCRIPTION
**Pull request by Galileo-Galilei**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by Galileo-Galilei** [comment]
This should be ready to merge.

**Comment by Galileo-Galilei** [comment]
@conda-forge-admin, please ping conda-forge/staged-recipes

Hi team, is there anything I can do to help this PR being reviewed?

**Comment by Galileo-Galilei** [comment]
@conda-forge-admin, please ping @conda-forge/staged-recipes

Hi team, sorry to insist!

**Comment by Galileo-Galilei** [comment]
@conda-forge-admin, please ping @conda-forge/staged-recipes

Hi team, i'll give another try!

**Comment by Galileo-Galilei** [comment]
@conda-forge-admin, please ping @conda-forge/staged-recipes

Hi team, i'll give another try!

File: LICENSE.txt [hunk 1]
```diff
@@ -24,4 +24,4 @@ DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
 SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
 CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
 OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
-OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
```
Comment (ocefpaf) [inline LICENSE.txt:27]: Can you revert this change in our license? I guess that is a newline?

File: recipes/kedro-mlflow/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  sha256: bad739aecfb1794a00e95dfc364731599b47c86effba6d8c88a2327bff0d8d62
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6, <3.9
```
Comment (ocefpaf) [inline recipes/kedro-mlflow/meta.yaml:19]:
```suggestion
    - python >=3.6,<3.9
```

File: recipes/kedro-mlflow/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    - pip
+  run:
+    - python >=3.6, <3.9
```
Comment (ocefpaf) [inline recipes/kedro-mlflow/meta.yaml:22]:
```suggestion
    - python >=3.6,<3.9
```

File: recipes/kedro-mlflow/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    - kedro >=0.17.1, <0.18.0
```
Comment (ocefpaf) [inline recipes/kedro-mlflow/meta.yaml:23]:
```suggestion
    - kedro >=0.17.1,<0.18.0
```

File: recipes/kedro-mlflow/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+    - mlflow >=1.0.0, <2.0.0
```
Comment (ocefpaf) [inline recipes/kedro-mlflow/meta.yaml:24]:
```suggestion
    - mlflow >=1.0.0,<2.0.0
```

File: recipes/kedro-mlflow/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+    - pydantic >=1.0.0, <2.0.0
```
Comment (ocefpaf) [inline recipes/kedro-mlflow/meta.yaml:25]:
```suggestion
    - pydantic >=1.0.0,<2.0.0
```

File: LICENSE.txt (hunk 1 above)
Comment (Galileo-Galilei) [reply LICENSE.txt]: Yes, it is a mistake.

**Comment by Galileo-Galilei** [comment]
Hi @ocefpaf, thank you very much for the review! I've made all requested changes, it should be ok now.

**Comment by Galileo-Galilei** [comment]
Hi @ocefpaf, any chances it could get merge ? Do you need me to do anything else? 
Thank you very much!
